src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -226,6 +237,10 @@ public function buildView(FormView $view, FormInterface $form, array $options)
      */
     public function finishView(FormView $view, FormInterface $form, array $options)
     {
+        if (in_array($options['widget'], array('text', 'hidden'), true)) {
+            return;
+        }
+
         if ($options['expanded']) {
```

Comment:
One last comment "for the road" if I may :

You did the change in https://github.com/bamarni/symfony/blob/9ba6a34f64fc063b3bfc8874d6f9300ba8f4c0be/src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php#L145

Why not this PR ?

Handling this everywhere in the class (except for choice view vars) is how you can make `$options['expanded']` deprecated otherwise it has to be marked internal.